### PR TITLE
Added test case for delete function

### PR DESCRIPTION
**Pull request by troublemagnet**

### CODE REVIEW

File: project_admin/views.py [hunk 1]
```diff
@@ -153,9 +153,8 @@ def delete_file(request, file_id):
 def update_file_metadata(metadata):
     files = FileMetaData.objects.all()
     for file in files:
-        file.name = metadata["file_{}_name".format(file.id)]
-        file.description = metadata["file_{}_description"
-                                    .format(file.id)]
-        file.tags = json.dumps(metadata["file_{}_tags"
-                               .format(file.id)].split(","))
+        file.name = str(metadata.get("file_1_name"))
```
Comment (sunidhiraheja) [inline project_admin/views.py]: We wouldn't want to hardcode this.

**Comment by jasmine1997** [comment]
Hey @troublemagnet, thanks for adding the test!
This looks good, with the exception of two lines in `views.py` where you hardcode the file ID, specifically lines 156 and 159. They should be using the `file_id` parameter like in line 158.

**Comment by troublemagnet** [comment]
Yeah, my bad. While tracing the errors, tried all possible things!! I'll fix soon.

**Comment by troublemagnet** [comment]
@jasmine1997 fixed :D

**Comment by jasmine1997** [comment]
@troublemagnet On a closer look, I noticed that you're not passing the form data in the delete call, which is causing the test to raise errors.
For why you need to do so, you can take a look at the discussion on https://github.com/gedankenstuecke/oh_data_uploader/pull/45.

Suggested change:

    response = c.post("/project-admin/delete-file/1/",
                      {'file_1_name': 'foo',
                       'file_1_description': 'bar',
                       'file_1_tags': 'my,tags,are,good'},
                       follow=True)

You can leave `update_file_metadata` unchanged.
Using `.get()` makes no difference since it just returns `None` instead of raising an error.

**Comment by troublemagnet** [comment]
Oh great, read the discussion. Thanks!

**Comment by gedankenstuecke** [comment]
Thanks for the great feedback @jasmine1997 and thanks for fixing it @troublemagnet! I'll merge!

**Pull request merged**